sqlite.zig

```diff
@@ -1089,6 +1100,15 @@ pub fn Statement(comptime opts: StatementOptions, comptime query: ParsedQuery) t
                         _ = c.sqlite3_bind_null(self.stmt, column);
                     },
                     .Null => _ = c.sqlite3_bind_null(self.stmt, column),
+                    .Enum => {
+                        if (comptime std.meta.trait.isZigString(FieldType.BaseType)) {
+                            return self.bindField(FieldType.BaseType, field_name, column, @tagName(field));
```

Comment:
this needs to pass `i` and not `column`, otherwise the subsequent `bindField` will try to bind into the wrong column (because we do `column = i + 1` above).